gbpservice/neutron/services/grouppolicy/drivers/oneconvergence_resource_mapping.py

```diff
@@ -0,0 +1,71 @@
+# One Convergence, Inc. CONFIDENTIAL
+# Copyright (c) 2012-2015, One Convergence, Inc., USA
+# All Rights Reserved.
+#
+# All information contained herein is, and remains the property of
+# One Convergence, Inc. and its suppliers, if any. The intellectual and
+# technical concepts contained herein are proprietary to One Convergence,
+# Inc. and its suppliers.
+#
+# Dissemination of this information or reproduction of this material is
+# strictly forbidden unless prior written permission is obtained from
+# One Convergence, Inc., USA
+
+
+import netaddr
```

Comment:
This and further unused imports have to be removed. That can be done in a subsequent commit.